src/main/java/io/jenkins/plugins/analysis/core/scm/BlameFactory.java

```diff
@@ -20,6 +22,20 @@
  * @author Lukas Krose
  */
 public final class BlameFactory {
+
+    private static JenkinsFacade jenkinsFacade = new JenkinsFacade();
+    private static GitChecker gitChecker = new GitChecker();
+
+    @VisibleForTesting
+    static void setJenkinsFacade(JenkinsFacade facade) {
```

Comment:
`final` for all parameters (see [coding style](https://github.com/uhafner/codingstyle))